int64 cone_lat
int64 cone_lon
---
geometry_msgs/PoseStamped cone_loc